### DISCUSSION TITLE
OpenAI Assistants API Integration Inquiry!

### DISCUSSION THREAD

**Discussion by adensattler** [MeSilicon7/LexiStreamKit#1] [Q&A]
Hey! I stumbled across this repo while working on a project of my own and managed to get it to stream responses from the OpenAI Assistants API which added streaming capability a few months ago. I would love to contribute to the project as it helped me out a ton! I'm more than happy to fork the repo and start a PR if you're interested in checking it out.

As far as I can tell there aren't any public projects that actually integrate the Assistants streaming functionality into a Flask web app (I found one or two in Next.js) which is pretty cool so hopefully it can actually help somebody else! Let me know what you think!

**Comment by MeSilicon7** [comment]
Feel free to fork the repository and initiate a pull request.
